dj_cqrs/delay.py

```diff
@@ -0,0 +1,63 @@
+#  Copyright © 2021 Ingram Micro Inc. All rights reserved.
+
+from datetime import timedelta
+from queue import PriorityQueue
+
+from dj_cqrs.utils import utc_now
+
+
+class DelayMessage:
+    """Delay message.
+
+    :param delivery_tag: The server-assigned and channel-specific delivery tag.
+    :type delivery_tag: int
+    :param payload: Transport payload.
+    :type payload: dj_cqrs.dataclasses.TransportPayload
+    """
+
+    def __init__(self, delivery_tag, payload, delay):
+        assert delay >= 0
+        self.delivery_tag = delivery_tag
+        self.payload = payload
+        self.eta = utc_now() + timedelta(seconds=delay)
```

Comment:
I think it is proper to pass `eta` inside to make user decide how to calculate it + in common - to hide `now` inside class of function is not a good idea